### ISSUE TITLE
[Feature Request] I would like to contribute

### ISSUE THREAD

**Issue by elisamuelps** [sillvva/rpg-schedule#184]
Hey! first of all thanks for making this discrod plugin, I just want to know if you have a rodmap in place of features that need to be implemented in this project, also I want to develop locally but you have mentioned in the changelog that you have moved from **heroku** to **aws**, I currently don't have experience with aws and I would like to know as well what is the new setup to run the bot locally.

Feature wise I would like to work on a small fix on the game _"run time"_ as by default it is in hours but I would like to add the option to be able to use hours or minutes as needed.

**Comment by elisamuelps** [comment]
Just to add to what I previously said, are you using react with the website? as I can help as well with the website itself, not a react expert but I can handle it, I do also have experience with vue.

**Comment by sillvva** [comment]
I am not using React. I'm using an EJS multi-page application. I've thought about switching to a single-page application, but the transition would be pretty big, so I've been putting it off. I do have experience in Angular, React, and Vue, though. So the thought is tempting.

The instructions to develop locally are included in the [README.md](https://github.com/sillvva/rpg-schedule/blob/development/README.md)

As far as upcoming features, one that is coming soon is an update to the monthly rescheduling. There's going to be an option to reschedule either on the Xth day of the month or based on Xth [weekday] of the month. (ex: 2nd Sunday of the month)

**Comment by elisamuelps** [comment]
Amazing! Thank you, I will get the repo, try and help with new features and send pull requests as neccesary, I will also try and build a vuejs fork of this repo to have a tentative move to vuejs. I will get in contact later to discuss more about it, but in the meantime I will get my hands dirty locally.

**Comment by sillvva** [comment]
Hey, just wanted to say thanks for looking into this and give a heads up that I have decided that for the sake of several upcoming features that it was the right time to make the switch to a SPA. Following your lead, I chose Nuxt (a Vue Framework). Here is the repo:

https://github.com/sillvva/rpg-schedule-nuxt

It is based entirely on the current version of the site, but with a fresh new look and some new features. Note that this is just a UI layer. The current repository will be serving as the bot and API layer.

Feel free to take a look and make any suggestions, fork, pull requests, etc. I plan to make it go live tomorrow evening.
